deploy-board/deploy_board/webapp/templatetags/utils.py

```diff
@@ -654,6 +658,8 @@ def hostButton(host):
 @register.filter("hostIcon")
 def hostIcon(host):
     if host['state'] == 'PROVISIONED':
+        if host["ip"] is None:
```

Comment:
What's `host["ip"]` of a new host? Is it not `None`?

Reply:
If a host is PROVISIONED by cloud-init fails it will have
1. hostId assigned to hostName  
2. No IP to connect to